DigitalLearningSolutions.Data.Tests/TestHelpers/TutorialContentTestHelper.cs

```diff
@@ -84,5 +85,16 @@ DELETE CustomisationTutorials
                 new { customisationId, tutorialId }
             );
         }
+
+        public int? GetAspProgressFromTutorialAndProgressId(int tutorialId, int progressId)

```

Comment:
It could easily go into the Data project, but it is only used for a single test currently, and I can't foresee any time it would need to be used outside of tests. Do we want it moved to the data project?

Reply:
Sorry, thought I'd deleted all the TODO comments 🤦 Yeah, it's fine in tests I think.